rs/tests/crypto/ingress_verification_test.rs

```diff
@@ -764,8 +771,10 @@ pub fn requests_to_mgmt_canister_with_delegations(env: TestEnv) {
                 let mut identities = vec![];
                 let mut targets = vec![];
 
-                for _ in 0..=delegation_count {
-                    let id_type = GenericIdentityType::random_incl_canister(&canister, rng);
+                identities.push(user.clone());
+
+                for _ in 1..=delegation_count {
+                    let id_type = GenericIdentityType::random(rng);
```

Comment:
Curious why canister identities are not used here. Not using them, however, is fine by me.

Reply:
No particular reason outside that it seemed like a complication with respect to the intent of this particular test. The test doc doesn't mention anything about using different auth types here, and the Haskell tests of the delegation logic used only Ed25519.